app/client/components/OverlayLoader.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { css } from "@emotion/core";
+import { textSans } from "@guardian/src-foundations/typography";
+import { Spinner } from "./spinner";
+
+interface OverlayLoaderProps {
+  message: string;
+}
+
+const overlay = css`
+  z-index: 10000;
+  display: block;
+  border: 0 none transparent;
```

Comment:
It doesn't look like this is doing anything?

Reply:
not too sure tbh, I copied these styles directly from Support. https://github.com/guardian/support-frontend/tree/main/support-frontend/assets/components/progressMessage